src/send-private-tx/send-private-tx.ts

```diff
@@ -0,0 +1,25 @@
+import { config } from 'dotenv'
+import { Producer, ProtocolType } from 'pegasys-orchestrate'
+
+// Load ENV variables
+config();
+
+export const sendPrivateTx = async () => {
+  const producer = new Producer(['localhost:9092']);
+  await producer.connect();
+
+  // Deploy a new Counter contract and returns the ID of the request
+  const requestId = await producer.sendTransaction({
```

Comment:
We used to have a commented out version of how to send Tessera private transaction here. I personally didn't like it much, probably it would be a good idea to have a separate example for Orion and tessera